inst/templates/BS4/content-authors.html

```diff
@@ -1,6 +1,7 @@
 <div class="row">
   <div class="contents col-md-9">
     {{#logo}}<img src="{{logo.src}}" class="pkg-logo" alt="" >{{/logo}}
+    <div class="kinda-section">
```

Comment:
What's the point of this class?

Reply:
oops, I had for one moment thought there was some css clash but there isn't.